core/src/main/java/jenkins/widgets/HistoryPageFilter.java

```diff
@@ -101,85 +105,116 @@ public void setSearchString(@Nonnull String searchString) {
     /**
      * Add build items to the History page.
      *
-     * @param items The items to be added. Assumes the list of items are in descending queue ID order i.e. newest first.
+     * @param runItems The items to be added. Assumes the items are in descending queue ID order i.e. newest first.
+     * @deprecated Replaced by add(Iterable&lt;T&gt;) as of version 2.15
      */
-    public void add(@Nonnull List<T> items) {
-        if (items.isEmpty()) {
+    @Deprecated
+    public void add(@Nonnull List<T> runItems) {
+        addInternal(runItems);
+    }
+
+    /**
+     * Add build items to the History page.
+     *
+     * @param runItems The items to be added. Assumes the items are in descending queue ID order i.e. newest first.
+     * @since 2.15
```

Comment:
I realize I was too vague in my earlier comment. We typically write `@since TODO` as sometimes PRs linger a bit (or a lot). Not a big deal, but FYI for the future.